tests/io_utils.go

```diff
@@ -117,11 +120,46 @@ func FixErrorDevice(nodeName string) {
 
 }
 
+func executeDeviceMapperOnNode(nodeName string, cmd []string) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	Expect(err).ToNot(HaveOccurred())
+
+	// Image that happens to have dmsetup
+	image := fmt.Sprintf("%s/virt-launcher:%s", flags.KubeVirtRepoPrefix, flags.KubeVirtVersionTag)
+	pod := &k8sv1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "device-mapper-pod-",
+		},
+		Spec: k8sv1.PodSpec{
+			RestartPolicy: k8sv1.RestartPolicyNever,
+			Containers: []k8sv1.Container{
+				{
+					Name:    "launcher",
+					Image:   image,
+					Command: cmd,
+					SecurityContext: &k8sv1.SecurityContext{
+						Privileged: pointer.BoolPtr(true),
+					},
+				},
+			},
+		},
+	}
+	pod, err = virtClient.CoreV1().Pods(util.NamespaceTestDefault).Create(context.Background(), pod, metav1.CreateOptions{})
+	Expect(err).ToNot(HaveOccurred())
+
+	Eventually(func() bool {
+		pod, err := virtClient.CoreV1().Pods(util.NamespaceTestDefault).Get(context.Background(), pod.Name, metav1.GetOptions{})
+		if err != nil {
+			return false
+		}
+		return pod.Status.Phase == k8sv1.PodSucceeded
+	}, 30*time.Second).Should(BeTrue())
+}
+
 func CreateFaultyDisk(nodeName, deviceName string) {
 	By(fmt.Sprintf("Creating faulty disk %s on %s node", deviceName, nodeName))
 	args := []string{"dmsetup", "create", deviceName, "--table", "0 204791 error"}
```

Comment:
Please add a validation command that will verify we have `dmsetup` on the pod something like `which dmsetup && ...`
Doing so will prevent flakes in case the image will change :)

Reply:
I'm not sure the error message for which is more clear than the error message for running dmsetup

Reply:
Sorry I think I got it wrong, since this is pod if the binary wont be there the pod will finish with phase `Failed` so we are good.